tests/test_e2e_11_mef_eline.py

```diff
@@ -125,31 +132,20 @@ def test_on_primary_path_fail_should_migrate_to_backup_with_dynamic_discovery_en
                 {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
                  "endpoint_b": {"id": "00:00:00:00:00:00:00:02:3"}}
             ],
-            "backup_path": [
-                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
-                 "endpoint_b": {"id": "00:00:00:00:00:00:00:04:4"}},
-                {"endpoint_a": {"id": "00:00:00:00:00:00:00:04:3"},
-                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:4"}},
-                {"endpoint_a": {"id": "00:00:00:00:00:00:00:03:1"},
-                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:4"}}
-            ],
-            "dynamic_backup_path": "true",
-            "active": "true",
-            "enabled": "true"
+            "dynamic_backup_path": True,
+            "active": True,
```

Comment:
active should be an internal attribute, should now allow change on the payload

Reply:
This is resolved with commit [25c6833](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/25c68330aa49b4fd9a01e478bffe9897d72d939e)